msos/msos.csproj

```diff
@@ -26,6 +25,7 @@
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
     <Prefer32Bit>true</Prefer32Bit>
     <UseVSHostingProcess>false</UseVSHostingProcess>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Now that you refactored a bunch of stuff, just double-checking -- do we still need to compile with `/unsafe`?